### PR TITLE
Find now

### LINKED ISSUE #85: Find Now functionality in HomePage
Search an NGO according to 3 filter parameters

### PR DESCRIPTION
**Pull request by fadime-ozdemir**
Add function for the find now button *it doesn't work properly because I can't pass state to siblings
Add click events for the other buttons: 

> More categories
> See more blogs
> Sign Up

Add space between buttons in the navbar
Align everything center according the design
Update tests

### CODE REVIEW

File: src/components/NavBar/index.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const NavBar = () => {
         <Navbar.Brand>{t("homePage.navbar.home")}</Navbar.Brand>
       </NavLink>
       <Nav className="ml-auto">
-        <NavLink to="/findNgo" className="nav-link">
+        <NavLink to="/findNgos" className="nav-link">
```
Comment (senzeynep) [inline src/components/NavBar/index.jsx:19]: Is it findNgo or findNow? And shouldn't it be only in Footer, not NavBar?

**Review by senzeynep — CHANGES_REQUESTED** [review]

File: src/components/NavBar/index.jsx (hunk 1 above)
Comment (fadime-ozdemir) [reply src/components/NavBar/index.jsx:19]: I need to change everything to findngos, it make more sense than findnow

**Comment by fadime-ozdemir** [comment]
PTAL

File: src/App.js [hunk 2]
```diff
@@ -24,30 +26,32 @@ function App() {
     { path: "/aboutus", name: "aboutus", Component: AboutUs },
     { path: "/aboutus", name: "aboutus", Component: AboutUS },
     { path: "/stories", name: "stories", Component: Stories },
-    { path: "/findnow", name: "findnow", Component: Resources },
+    { path: "/findNgos", name: "findngos", Component: Resources },
```
Comment (halitbatur) [inline src/App.js]: I think the name should be the same as the path ?

File: src/components/Header/index.jsx [hunk 3]
```diff
@@ -1,49 +1,53 @@
...
+import { useDispatch, useSelector } from "react-redux";
 
 const Header = () => {
+  const dispatch = useDispatch();
+  const searchInput = useSelector((state) => state.currentSearchWordsReducer);
   const { t } = useTranslation();
+  const history = useHistory();
+  const [input, setInput] = useState("");
+
+  console.log(searchInput);
```
Comment (halitbatur) [inline src/components/Header/index.jsx]: lets remove the console.log

File: src/containers/Resources/index.jsx [hunk 4]
```diff
@@ -4,12 +4,16 @@ import NgoList from "../../components/NgoList";
 import AdvancedFilters from "../../components/AdvancedFilters";
 import PagesList from "../../components/PagesList";
 import database from "../../firebaseConfig";
+// import { wordsSearchedAction } from "../../redux/action/index";
```
Comment (halitbatur) [inline src/containers/Resources/index.jsx]: no commented code please.

File: src/containers/Resources/index.jsx [hunk 5]
```diff
@@ -4,12 +4,16 @@ import NgoList from "../../components/NgoList";
...
+import { useSelector } from "react-redux";
 
 const Resources = () => {
   const [allNgos, setAllNgos] = useState([]);
   const [filteredNgos, setFilteredNgos] = useState([]);
   const [filterOptions, setFilterOptions] = useState([]);
-  const [searchKey, setSearchKey] = useState("");
+  // const dispatch = useDispatch();
```
Comment (halitbatur) [inline src/containers/Resources/index.jsx]: commented code

**Review by halitbatur — APPROVED** [review]
few small changes and its ready to be merged

**Comment by fadime-ozdemir** [comment]
PTAL

**Review by halitbatur — APPROVED** [review]

**Pull request merged**